MQTTClient.py

```diff
@@ -63,8 +69,9 @@ def _on_message(self, client, userdata, message):
     def publish(self, task):
         topic = "%s/%s/%s/R/%s" % (self.mqttDataPrefix, task['family'], task['deviceId'], task['param'])
         try:
-            self._mqttConn.publish(topic, payload=task['payload'])
             self.logger.debug('Sending:%s' % (task))
+            #self._mqttConn.publish(topic, payload=task['payload'])
+            publish.single(topic, payload=task['payload'], hostname=self.host, auth=self.auth, port=self.port)
```

Comment:
I moved the debug message BEFORE the publish call, because if the publish call fails, the debug message would never be printed...